examples/post-sync/controller.go

```diff
@@ -0,0 +1,312 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/go-logr/logr"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"kpt.dev/configsync/pkg/api/configsync/v1beta1"
+)
+
+// Sync kinds
+const (
+	RootSyncKind = "RootSync"
```

Comment:
looks like we're still not linting this directory or something. otherwise it would have failed on these. Can you enable linting and add const descriptions?

Reply:
added to lint, plus the unit test to config sync presubmit.

Reply:
lint didn't error that these public constants weren't documented???

Reply:
Not really.. It uses the golangci.yaml config in the post-sync/ dir though. I'll add comments

Reply:
Comment added.